configs/client2/ip.sh

```diff
@@ -3,9 +3,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 # Add IPv4
-ip address add 172.17.22.2/24 dev eth1
-ip route add 172.17.0.0/16 via 172.17.22.1
+ip address add 172.17.0.22/24 dev eth1
```

Comment:
I think it is safe to remove those client.sh files, since this is more easy to set via `exec` in the clab file?

Reply:
ah. I see you use this script in exec, all good then
so no routes are needed?